workers/unity/Packages/com.improbable.gdk.core/CodegenAdapters/ComponentReplicationHandler.cs

```diff
@@ -7,10 +7,14 @@ namespace Improbable.Gdk.Core.CodegenAdapters
     public abstract class ComponentReplicationHandler
     {
         public abstract uint ComponentId { get; }
-        public abstract ComponentType[] ReplicationComponentTypes { get; }
+        public abstract EntityArchetypeQuery ComponentUpdateQuery { get; }
+        public abstract EntityArchetypeQuery[] CommandQueries { get; }
 
-        public abstract void ExecuteReplication(ComponentGroup replicationGroup, Connection connection);
-        public abstract void SendCommands(SpatialOSSendSystem sendSystem, Connection connection);
+        public abstract void ExecuteReplication(ComponentGroup replicationGroup, ComponentSystemBase system,
+            Connection connection);
+
+        public abstract void SendCommands(ComponentGroup commandGroup, SpatialOSSendSystem sendSystem,
```

Comment:
Why does this one require `SpatialOSSendSystem` rather than `ComponentSystemBase`?

Reply:
Because I didn't write this one, just changed it. Will change this too. (ComponentSystemBase is better as it also includes job systems)